Add before-afterScript warning regarding containers (#3167)
nextflow-io/nextflow: docs/process.rst

The ``afterScript`` directive allows you to execute a custom (Bash) snippet immediately *after* the main process has run.
This may be useful to clean up your staging area.

> .. warning:: When combined with the :ref:`container directive <process-container>`, the ``afterScript`` will be 

Reviewer: ```suggestion
.. note:: When combined with the :ref:`container directive <process-container>`, the ``afterScript`` will be 
```

      """
    }

> .. warning:: When combined with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 

Reviewer: ```suggestion
.. note:: When combined with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 
```

Better to use `note` instead of a `warning`

> .. warning:: In combination with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 
>    executed outside the specified container. The ``beforeScript`` is always executed in the host environment.

Reviewer 2: ```suggestion
.. warning:: When combined with the :ref:`container directive <process-container>`, the ``beforeScript`` will be 
   executed outside the specified container. In other words, the ``beforeScript`` is always executed in the host environment.
```
